docs/site-example/site.conf

```diff
@@ -65,6 +71,9 @@
     ap = {
       ssid = 'alpha-centauri.freifunk.net',
     },
+    owe = {
```

Comment:
How about making this an example for a backwards compatible (`transition_mode = true,`) OWE network?

Reply:
The transition-mode is commented out in the 2.4GHz part, as it's an optional setting. These options do not seem to be included in the `wifi5` section, whus I've decided to skip it here.